server/controllers/stripe.js

```diff
@@ -8,6 +11,50 @@ exports.createUser = asyncHandler(async (req, res, next) => {
   const user = await User.findById(req.user.id);
   const profile = await Profile.findOne(req.user);
 
+  const schedule = await Schedule.create({
+    name: 'default schedule',
+    periods: {
+      monday: {
+        startTime: '10',
+        endTime: '22',
+      },
+      tuesday: {
+        startTime: '10',
+        endTime: '22',
+      },
+      wednesday: {
+        startTime: '10',
+        endTime: '22',
+      },
+      thursday: {
+        startTime: '10',
+        endTime: '22',
+      },
+      friday: {
+        startTime: '10',
+        endTime: '22',
+      },
+      saturday: {
+        startTime: '10',
+        endTime: '22',
+      },
+      sunday: {
+        startTime: '10',
+        endTime: '22',
+      },
+    }
+  });
+
+  const availability = await Availability.create({
+    scheduleIds: [schedule._id],
+    activeScheduleId: schedule._id,
+  });
+
+  const petsitter = await PetSitter.hydrate(profile.toObject());
+  petsitter.kind = 'PetSitter';
+  petsitter.availabilityId = availability._id;
+  await petsitter.save()
```

Comment:
I think we should remove this too since we have work being done on signing up as a pet sitter

Reply:
removed in latest version